src/day-1-toy/dicts.py

```diff
@@ -25,5 +25,21 @@
 ]
 
 # Write a loop that prints out all the field values for all the waypoints
+# for i in waypoints:
+#     for key,val in i.items():
+#         print key, "=>", val
+
 
 # Add a new waypoint to the list
+
+waypoints.append({
+    "lat": 88,
+        "lon": -167,
+        "name": "a fourth place"
+})
+
+# print(waypoints)
+
+for i in waypoints:
+    for key,val in i.items():
+        print key, "=>", val
```

Comment:
If the above `print` statement works, you are probably using Python 2.x. This is OK for now, but there might be turbulence later on, as future projects may require Python 3.x. Let me know about your environment set-up when you have the time.

Reply:
Hey, `items()` is Python 3.  Derived from `viewitems()` in Python 2.

Reply:
🤔 Then why is that print statement working? Thought it shouldn't work. 🔎 An opportunity for investigation.